.travis.yml

```diff
@@ -1,28 +1,44 @@
-## Inspired by https://gist.github.com/y0ssar1an/df2dab474520c4086926f672c52db139
-
 language: go
+cache: false
 
 go:
   - 1.13.x
 
-# Only clone the most recent commit.
 git:
-  depth: 1
-
-# Anything in before_script that returns a nonzero exit code will flunk the
-# build and immediately stop. It's sorta like having set -e enabled in bash.
-# Make sure you've pinned the version of golangci-lint by running this command
-# in your project directory:
-#   GO111MODULE=on go get github.com/golangci/golangci-lint@v1.21.0
-# You should see this line in your go.mod file:
-#   github.com/golangci/golangci-lint v1.21.0
+  depth: 10
```

Comment:
Curious, why depth 10?

Reply:
I never used this property before, so I preserved it given that it was already here, but increased so that we can have multiple folks contributing. When it was `1`, Travis was flipping and restarting the run as people pushed commits, wasn't able to see one go through.